pkgs/tools/virtualization/linode-cli/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq yq-go
+
+set -eou pipefail
+
+cd "$(dirname "$0")"
+
+SPEC_VERSION=$(curl -s https://www.linode.com/docs/api/openapi.yaml | yq eval '.info.version' -)
+
+SPEC_SHA256=$(nix-prefetch-url --quiet https://raw.githubusercontent.com/linode/linode-api-docs/v${SPEC_VERSION}/openapi.yaml)
+
+LATEST_VERSION=$(curl -s -u ":${GITHUB_TOKEN}" \
```

Comment:
I don't think it is good to require GITHUB_TOKEN to run the update script.

For example, I just tried to test this and:

```ShellSession
$ nix-shell maintainers/scripts/update.nix --argstr path linode-cli
this derivation will be built:
  /nix/store/wg9851dn1nkipy5f881z0b7yg7687b1i-packages.json.drv
building '/nix/store/wg9851dn1nkipy5f881z0b7yg7687b1i-packages.json.drv'...

Going to be running update for following packages:
 - linode-cli-5.9.0

Press Enter key to continue...

Running update for:
 - linode-cli-5.9.0: UPDATING ...
 - linode-cli-5.9.0: ERROR

--- SHOWING ERROR LOG FOR linode-cli-5.9.0 ----------------------

these 7 paths will be fetched (1.88 MiB download, 6.74 MiB unpacked):
  /nix/store/0ndidfcysc04bi9yg6wd2y41nb3cqcbk-libssh2-1.9.0-dev
  /nix/store/4jz8r4ki4yf0920ll3ldywv4j8qin12k-libidn-1.38-bin
  /nix/store/7f4c38d8dhgzb5a2xz6ik0wg9x483l5f-yq-go-4.13.0
  /nix/store/f4cfw68gkf3yl65npd7ljvb82m52ih9j-nghttp2-1.43.0-dev
  /nix/store/j5wjmhgcwash0awgm56hz5aciyrv57qj-curl-7.76.1-dev
  /nix/store/s710cghdniqaam6a8v5djr2gh1wmv6dp-libidn-1.38-dev
  /nix/store/y5vr0g32n9sf8qk9mf0vkzkf9ggc4v54-jq-1.6-dev
copying path '/nix/store/y5vr0g32n9sf8qk9mf0vkzkf9ggc4v54-jq-1.6-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/7f4c38d8dhgzb5a2xz6ik0wg9x483l5f-yq-go-4.13.0' from 'https://cache.nixos.org'...
copying path '/nix/store/0ndidfcysc04bi9yg6wd2y41nb3cqcbk-libssh2-1.9.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/f4cfw68gkf3yl65npd7ljvb82m52ih9j-nghttp2-1.43.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/4jz8r4ki4yf0920ll3ldywv4j8qin12k-libidn-1.38-bin' from 'https://cache.nixos.org'...
copying path '/nix/store/s710cghdniqaam6a8v5djr2gh1wmv6dp-libidn-1.38-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/j5wjmhgcwash0awgm56hz5aciyrv57qj-curl-7.76.1-dev' from 'https://cache.nixos.org'...
/home/ryantm/p/nixpkgs/pkgs/tools/virtualization/linode-cli/update.sh: line 19: GITHUB_TOKEN: unbound variable


--- SHOWING ERROR LOG FOR linode-cli-5.9.0 ----------------------

Packages updated!
```

Reply:
If the variable is blank it could be ommited then.

Reply:
Done.